requirements_dev.txt

```diff
@@ -2,3 +2,5 @@ pytest>=7.2.0
 pytest-cov>=4.0.0
 pytest-mock>=3.10.0
 typing_extensions>=4.4.0
+
+reno[sphinx]==3.3.0
```

Comment:
This can probably be loosened to `~3.3` or `>=3.3`, or even move up the lower bound to `>=4.0`. Is the `sphinx` extra needed here at all since (AFAIU) the docs don't use reno anymore?